packages/vue-app/template/App.js

```diff
@@ -64,7 +64,7 @@ export default {
   },
   created() {
     // Add this.$nuxt in child instances
-    Vue.prototype.<%= globals.nuxt %> = this
+    Vue.prototype.<%= globals.nuxt %> = <%= (globals.nuxt !== '$nuxt' ? 'Vue.prototype.$nuxt = ' : '') %>this
```

Comment:
I think `globals` is for hiding `$nuxt` actually, so maybe change `nuxt-link.client.js` to use dynamic `globals.nuxt` is better 😃